docs/resources/Audit_Log.md

```diff
@@ -0,0 +1,143 @@
+| CHANNEL_OVERWRITE_DELETE | 15 |
+| MEMBER_KICK | 20 |
+| MEMBER_PRUNE | 21 |
+| MEMBER_BAN_ADD | 22 |
+| MEMBER_BAN_REMOVE | 23 |
+| MEMBER_UPDATE | 24 |
+| MEMBER_ROLE_UPDATE | 25 |
+| ROLE_CREATE | 30 |
+| ROLE_UPDATE | 31 |
+| ROLE_DELETE | 32 |
+| INVITE_CREATE | 40 |
+| INVITE_UPDATE | 41 |
+| INVITE_DELETE | 42 |
+| WEBHOOK_CREATE | 50 |
+| WEBHOOK_UPDATE | 51 |
+| WEBHOOK_DELETE | 52 |
+| EMOJI_CREATE | 60 |
+| EMOJI_UPDATE | 61 |
+| EMOJI_DELETE | 62 |
+| MESSAGE_DELETE | 72 |
+
+###### Optional Audit Entry Info
+
+| Field | Type | Description | Action Type |
+| ----- | ---- | ----------- | ----------- |
+| delete_member_days | string | number of days after which inactive members were kicked | 21
+| members_removed | string | number of members removed by the prune | 21
+| channel_id | snowflake | channel in which the messages were deleted | 72
+| count | string | number of deleted messages | 72
+| id | snowflake | id of the overwritten entity | 13, 14, 15
+| type | string | type of overwritten entity ("member" or "role") | 13, 14, 15
+| role_name | string | name of the role if type is "role"| 13, 14, 15
+
+### Audit Log Change Object
+
+###### Audit Log Change Structure
+
+| Field | Type | Description |
+|-------|------|-------------|
+| new_value | [any](#DOCS_AUDIT_LOG/audit-log-change-key) | new value of the key |
+| old_value | [any](#DOCS_AUDIT_LOG/audit-log-change-key) | old value of the key |
+| key | string | type of audit log [change key](#DOCS_AUDIT_LOG/audit-log-change-key)
+
+###### Audit Log Change Key
+
+| Name | Object Changed | Type | Description |
+| ---- | ------ | ---------- | ----------- |
+| name | [guild](#DOCS_GUILD/guild-object)| string | name changed
+| icon_hash | [guild](#DOCS_GUILD/guild-object) | string | icon changed
+| splash_hash | [guild](#DOCS_GUILD/guild-object) | string | invite splash page artwork changed
+| owner_id | [guild](#DOCS_GUILD/guild-object) | snowflake | owner changed
+| region | [guild](#DOCS_GUILD/guild-object) | string | region changed
+| afk_channel_id | [guild](#DOCS_GUILD/guild-object) | snowflake | afk channel changed
+| afk_timeout | [guild](#DOCS_GUILD/guild-object) | int | afk timout duration changed
+| mfa_level | [guild](#DOCS_GUILD/guild-object) | int | two-factor auth requirement changed
+| verification_level | [guild](#DOCS_GUILD/guild-object) | int | required verification level changed
+| explicit_content_filter | [guild](#DOCS_GUILD/guild-object) | int | change in [whose messages](#DOCS_GUILD/explicit-content-filter-level) are scanned and deleted for explicit content in the server
+| default_message_notifications | [guild](#DOCS_GUILD/guild-object) | int | default [message notification level](#DOCS_GUILD/default-message-notification-level) changed|
+| vanity_url_code | [guild](#DOCS_GUILD/guild-object) | string | guild invite vanity url changed
+| $add | [guild](#DOCS_GUILD/guild-object) | [role](#DOCS_PERMISSIONS/role-object) object| new role added
```

Comment:
The value sent for the change keys `$add` and `$remove` is `[{role_obj}]` not just `{role_obj}`. Unless that has changed, should probably be clarified.